### PR TITLE
gradle+docs+tests: added gradle with build and tests, updated README.md

### PR DESCRIPTION
**Pull request by willeponken**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+## Test den perfekte applikation
+### *nix
+```
+./gradlew test
+```
+
+### Windoze
+```
+./gradlew.bat test
```
Comment (Hjortsberg) [inline README.md]: I think it went a bit too fast here, windows does not use the ./ for bat files! =)

Comment (willeponken) [reply README.md]: Lol nope you're right yes hehe thx bai

**Comment by willeponken** [comment]
Sorry @Hjortsberg I forgot to sign, please re-review :)

**Comment by willeponken** [comment]
@Hjortsberg Okay, THIS TIME IT SHOULD WORK YES

**Pull request merged**